src/assets/i18n/zh-hans.json

```diff
@@ -1,129 +1,129 @@
+  "Allow non-root mount": "允许非root挂载",
+  "Allow this replication to send large data blocks. The  destination system must also support large blocks. This setting cannot be changed  after it has been enabled and the replication task is created. For more details, see  <a href=\"https://linux.die.net/man/8/zfs\"  target=\"_blank\">zfs(8)</a>.": "允许此副本发送大数据块。 目标系统必须支持大数据块。 此设置在创建副本并启用后无法更改。 参阅  <a href=\"https://linux.die.net/man/8/zfs\"  target=\"_blank\">zfs(8)</a>.以获取更多信息",
+  "Allow {activities}": "允许 {activities}",
+  "Appends a suffix to the share connection path.  This is used to provide unique shares on a per-user, per-computer, or per-IP address basis.  Suffixes can contain a macro. See the  <a href=\"https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html\" target=\"_blank\">smb.conf</a> manual page for  a list of supported macros. The connectpath **must** be preset before a client connects.": "向共享连接路径添加一个前缀。  此操作用于提供基于每个用户，每台计算机，或者每个IP地址的个性化共享。前缀可以包含宏。 参阅  <a href=\"https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html\" target=\"_blank\">smb.conf</a> 以获取可用宏的清单。 连接路径 **必须** 在被客户端连接前设置。",
+  "Attach screenshots": "添加截屏",
+  "Back up the encryption key now! If the key is lost, the data on the disks will also be lost with no hope of recovery. Click <b>Download Encryption Key</b> to begin the download. This type of encryption is for users storing sensitive data. iXsystems, Inc. cannot be held responsible for any lost or unrecoverable data as a consequence of using this feature.": "请立刻备份加密密钥！如果密钥丢失，磁盘上的数据将一并丢失且不可恢复。点击 <b>下载密钥</b> 以开始下载。 此类加密用于用户存储敏感数据。iXsystems, Inc. 不对使用此功能造成的任何数据丢失负责。",
+  "Body": "部分",
+  "Browse to the path to be copied. Linux file path limits apply. Other operating systems can have different limits which might affect how they can be used as sources or destinations.": "浏览需要复制的路径。Linux文件路径限制应用。其他操作系统也可能包含限制其作为源或终点的限制。",
+  "Check Alerts for more details.": "查看警报以获取更多详细信息。",
+  "Click for information on    <a href=\"https://www.truenas.com/docs/truenasupgrades/\" target=\"_blank\">TrueNAS SCALE Migration, Nightly trains    and other upgrade options.</a>": "点击 <a href=\"https://www.truenas.com/docs/truenasupgrades/\" target=\"_blank\">以获取关于TrueNAS SCALE迁移，每夜更新通道以及更多升级选项.</a>",
+  "Clone To New Dataset": "克隆到新数据集",
+  "Close expanded row": "关闭扩展行",
+  "Close panel": "关闭面板",
+  "Collapse Row": "解散行",
+  "Congratulations! Your ticket has been submitted successfully. It may take some time before images appear.": "恭喜！您的工单已成功提交。 可能需要一些时间才能看到镜像。",
```

Comment:
The last part needs a bit of improvement. According to Google Translate, it says "It may take some time to see the mirror" which is confusing. Let's fix this. The rest of the translation looks good! Thank you!

Reply:
OK! Actually, "image" means "镜像" in Chinese , while "mirror" is "镜子"

Reply:
> The last part needs a bit of improvement. According to Google Translate, it says "It may take some time to see the mirror" which is confusing. Let's fix this. The rest of the translation looks good! Thank you!

OK! Actually, "image" means "镜像" in Chinese , while "mirror" is "镜子"